com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -4,40 +4,67 @@
 
 namespace UnityEngine.Rendering.HighDefinition
 {
+    /// <summary>
+    /// Probe Volume Evaluation Mode.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ProbeVolumesEvaluationModes
     {
+        /// <summary>Probe volumes are disabled.</summary>
         Disabled = 0,
+        /// <summary>Probe volumes are evaluated in the light loop.</summary>
         LightLoop = 1,
+        /// <summary>Probe volumes are evaluated in the material pass.</summary>
         MaterialPass = 2,
     }
 
+    /// <summary>
+    /// Encoding of Probe Volumes.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ProbeVolumesEncodingModes
     {
+        /// <summary>Encoded using L0 spherical harmonics.</summary>
         SphericalHarmonicsL0 = 0,
+        /// <summary>Encoded using L1 spherical harmonics.</summary>
         SphericalHarmonicsL1 = 1,
+        /// <summary>Encoded using L2 spherical harmonics.</summary>
         SphericalHarmonicsL2 = 2
     }
 
+    /// <summary>
+    /// Probe Volume bilateral filtering mode.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ProbeVolumesBilateralFilteringModes
     {
+        /// <summary>Bilateral filtering is disabled.</summary>
         Disabled = 0,
+        /// <summary>Bilateral filtering using validity.</summary>
         Validity = 1,
+        /// <summary>Bilateral filtering using octahedral depth.</summary>
         OctahedralDepth = 2
     }
 
+    /// <summary>
+    /// Project wide shader configuration options.
+    /// This enum will generate the proper shader defines.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ShaderOptions
     {
-        ColoredShadow = 1, // Allow to defined if colored shadow are supported in shaders or not
-        CameraRelativeRendering = 1, // Rendering sets the origin of the world to the position of the primary (scene view) camera
+        /// <summary>Support for colored shadows.</summary>
+        ColoredShadow = 1,
+        /// <summary>Use camera relative rendering to enhance precision.</summary>
```

Comment:
Uses [camera relative rendering](../manual/Camera-Relative-Rendering.md) to enhance precision.